app/src/main/java/com/kickstarter/viewmodels/PledgeFragmentViewModel.kt

```diff
@@ -1361,6 +1361,14 @@ interface PledgeFragmentViewModel {
                     this.lake.trackUpdatePledgePageViewed(it.first, it.second)
                 }
 
+            checkoutAndPledgeData
```

Comment:
This added code is duplicating the one on lines above it.
<img width="930" alt="Screen Shot 2021-04-05 at 1 37 31 PM" src="https://user-images.githubusercontent.com/4083656/113624155-20477b80-9614-11eb-86cd-0aae4a026c4d.png">

Reply:
done